test/e2e-common.sh

```diff
@@ -60,9 +60,16 @@ readonly BUCKETS=10
 LATEST_SERVING_RELEASE_VERSION=$(latest_version)
 
 # Latest net-istio release.
-LATEST_NET_ISTIO_RELEASE_VERSION=$(
-  curl -L --silent "https://api.github.com/repos/knative/net-istio/releases" | \
-    jq -r '[.[].tag_name] | sort_by( sub("knative-";"") | sub("v";"") | split(".") | map(tonumber) ) | reverse[0]')
+LATEST_NET_ISTIO_RELEASE_VERSION=$(align_net-istio_version "$LATEST_SERVING_RELEASE_VERSION")
+
+# Receives the latest serving version and searches for the same version with major and minor and searches for the latest patch
+function align_net-istio_version() {
```

Comment:
just a nit with the name

```suggestion
function latest_net_istio_version() {
```